textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslatorTest.java

```diff
@@ -10,6 +12,16 @@
 
 public class RegexToGramQueryTranslatorTest {
 	
+	private void printTranslatorResult(String regex) {
+		GramBooleanQuery exactQuery = RegexToGramQueryTranslator.translate(regex);
+		
+		System.out.println("regex: "+regex);
```

Comment:
We can't "assert" in many test cases because the expected query is too complex to build by hard code. So ( for now) in those test cases, we print out the query tree, and examine whether they are correct by hand...